### DISCUSSION TITLE
Following BlueSky account from Mastodon

### DISCUSSION THREAD

**Discussion by myfta** [snarfed/bridgy-fed#1299] [Q&A]
I'm all connected up and I can see my user page at https://fed.brid.gy/
It's not clear how to actually follow a user on BlueSky. Does my bridged acoount on BlueSky need to be a real account and do I need to log in there? If so, how's that done with no password?
Or do I follow BlueSky users from Mastodon, but surely they need to be bridged first? So how does that happen?

**Comment by qazmlp** [comment]
> Or do I follow BlueSky users from Mastodon, but surely they need to be bridged first? So how does that happen?

It's like this, yes. They need to follow the account `@ap.brid.gy` first to enable bridging. Afterwards, you can find them at `@…@bsky.brid.gy` where `…` is their bluesky handle.

If they have Chat open to anyone (**not** the default), I think you can also DM the bridge bot on Mastodon to send a request once.

**Reply by snarfed** [reply]
Thanks @qazmlp! More background in the docs: https://fed.brid.gy/docs#fediverse-follow-bluesky

**Reply by myfta** [reply]
> I think you can also DM the bridge bot on Mastodon to send a request once.

Does someone know how this DM function works? How do you DM a BlueSky user from Mastodon?
Thanks.

**Reply by TomCasavant** [reply]
> > I think you can also DM the bridge bot on Mastodon to send a request once.
> 
> Does someone know how this DM function works? How do you DM a BlueSky user from Mastodon? Thanks.

I don't think it's possible to DM a bluesky user from mastodon (they handle DMs differently and right now don't have a concept of private mentions like mastodon does), they were just saying to DM the mastodon bot directly (fed.brid.gy@fed.brid.gy) with the handle of the person you want to follow and it will ask the bluesky user to opt-in (though I haven't heard if this is working yet?)

**Reply by snarfed** [reply]
Right! I haven't documented or announced the DM prompt feature yet, but it should be working for fediverse => Bluesky. From a fediverse account, DM a Bluesky user's handle to @bsky.brid.gy@bsky.brid.gy, and BF will send a chat message to that Bluesky user saying that you've requested them to bridge their account.

It's almost done for Bluesky => fediverse too, but not fully shipped yet.

**Reply by myfta** [reply]
Do you see these requests? I have had no response either from @bsky.brid.gy or the BlueSky account.

**Reply by snarfed** [reply]
@myfta which fediverse account did you DM @bsky.brid.gy from?

**Reply by snarfed** [reply]
Thanks, I see it now. Looks like the leading `@` character was tripping us up. I've deployed a fix for that, feel free to try again.

**Comment by myfta** [comment]
So, has it gone through OK this time?

Graham

**Reply by snarfed** [reply]
Did you resend? I'm not seeing any new DMs from your account since the one you sent on 9/3.

**Comment by myfta** [comment]
Somehow I think my Mastodon app was not recognising @
***@***.*** as a valid account. I have tried it again.

**Reply by snarfed** [reply]
Huh, Bridgy Fed still hasn't received it.

**Comment by myfta** [comment]
Can you clarify the correct full Mastodon address for 'brid.gy', please.

**Reply by snarfed** [reply]
Sure! `@bsky.brid.gy@bsky.brid.gy` , https://fed.brid.gy/docs#fediverse-get-started . You've successfully sent at least one DM to it before, at 2024-09-03 17:52:43 UTC.

**Reply by snarfed** [reply]
Got one from you! 21:05:51 UTC today, looks like it worked.
